### DISCUSSION TITLE
Why the relicense?

### DISCUSSION THREAD

**Discussion by 8bitprodigy** [DescentDevelopers/Descent3#269] [General]
The MIT license was great. Why change it to GPL?

**Comment by Jayman2000** [comment]
This repo wasn’t actually relicensed. Here’s what happened:

- Kevin Bentley got permission from Parallax Software to release the source code for Descent 3, but Parallax Software didn’t say “you can release it under this particular license.” They just said “you can release it.”
- Kevin Bentley created a new GitHub repo. GitHub offered to automatically include a copy of the MIT License in the repo, and Kevin unintentionally agreed. This was an accident. (See 637df31ca1e47371fb8ca822f928734c13afd8cf and [this comment](https://github.com/DescentDevelopers/Descent3/issues/32#issuecomment-2061461160)).
- Later, Kevin realized the mistake and removed the copy of the MIT License. (See 6a7a14159a80c7c7683e75573dc0094ba96ad0c7).
- After talking with Parallax Software, Kevin got permission to release the code under the GPL (See 1f451632485491dfe5e5add498e6e2428107a147 and [these](https://github.com/DescentDevelopers/Descent3/issues/32#issuecomment-2066796122) [two](https://github.com/DescentDevelopers/Descent3/issues/32#issuecomment-2067716480) comments).

---

Why do you think that the MIT License would be a better choice than the GPL?
